app/serializers/keyword_serializer.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class KeywordSerializer
+  include JSONAPI::Serializer
+
+  attributes :name
```

Comment:
How about to pass created at attribute also?
```suggestion
  attributes :name, :created_at
```

Reply:
Makes sense! (Y)

Reply:
API documentation also updated accordingly ;-)